.github/workflows/ci.yaml

```
@@ -9,6 +9,24 @@ on:
             - main
 
 jobs:
+    benchmarks:
+        runs-on: ubuntu-latest
+        env:
+            CARGO_BUILD_JOBS: 2
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)